### ISSUE TITLE
Notify developers when devcontainer is changed to rebuild workspace

### ISSUE THREAD

**Issue by bpmct** [coder/envbuilder#125]
We want to add [notifications](https://www.notion.so/coderhq/Developer-Notifications-d209a32d178a473a9de3e5779d12a3fe?pvs=4) into our product. We should wait until this is complete before announcing v1.0 and leverage it to notify users when the devcontainer changed and prompt them to rebuild their workspace.

**Comment by johnstcn** [comment]
This may need an RFC to flesh out exactly how we want to handle the envbuilder -> coder integration, given that we want to avoid direct integrations when possible.

**Comment by bpmct** [comment]
yeah, let's pause on this until we have generic notifications support
